### DISCUSSION TITLE
Exclude photos from Archive

### DISCUSSION THREAD

**Discussion by pelinoleg** [damongolding/immich-kiosk#29] [Ideas]
Could an option be added to exclude photos from the Archive? And if we want to make it more complex, maybe even add the ability to exclude certain albums and people.

**Comment by lukaszwawrzyk** [comment]
+1 on excluding albums. I am mostly interested in showing all albums including these shared with me, except a few of them. Currently I need to list all albums and add new ones to the list. I think `KIOSK_ALBUM=all KIOSK_ALBUM_EXCLUDE=id1,id2` would be a nice solution to this.

Additionally it would be nice to be able to show timeline + all albums except excluded ones, but maybe this is too complex.

**Reply by damongolding** [reply]
Added in https://github.com/damongolding/immich-kiosk/pull/203 :)
